### PR TITLE
[1.10.x] fix double click prevention when client validation

### PR DESCRIPTION
**Pull request by jtkech**
If you click on submit but the form is not submitted because of client validation, then you can't submit anymore.

### CODE REVIEW

File: src/Orchard.Web/Themes/TheAdmin/Scripts/admin.js [hunk 1]
```diff
@@ -84,11 +84,14 @@
     //Prevent double-click on buttons of type "submit"
     $("form button[type='submit'], form input[type='submit']").click(function (e) {
         var form = $(this).closest("form")[0];
-        if (typeof(form.formSubmitted) != "undefined") {
+        if (typeof (form.formSubmitted) != "undefined" && form.formSubmitted == true) {
             e.preventDefault();
             return;
         }
         form.formSubmitted = true;
+        setTimeout(function () {
```
Comment (sebastienros) [inline src/Orchard.Web/Themes/TheAdmin/Scripts/admin.js]: How does it work ?

**Comment by sebastienros** [comment]
Fixes https://github.com/OrchardCMS/Orchard/pull/6766

**Comment by jtkech** [comment]
As i've tested, it releases the formSubmitted lock (implemented here) after 1 second.

So, if you click i.e `Publish` and you are still in the same edit form because of a client validation checking, the you can correct your input and retry a publish action.

**Comment by sebastienros** [comment]
Isn't there a way to not disable the button if the form is not submitted? Not sure if you are an expert at JAvascript, we might have to ask one's opinion ...

**Comment by jtkech** [comment]
I agree

**Comment by sebastienros** [comment]
I am currently working on it getting some tips from gurus.

**Comment by jtkech** [comment]
I am interested to know the final solution. Here i just fixed the current implementation, maybe we have better to use the `submit event` directly.

**Comment by jtkech** [comment]
@sebastienros 

Just tried with the submit event, seems to work by replacing this:

    $("form button[type='submit'], form input[type='submit']").click(function (e) {

with:

    $("form").submit(function (e) {

And now without the `setTimeout()` call.

Best.

**Comment by jtkech** [comment]
Just by curiosity i've done additional tests, then i let you investigate.

So the solution to use the submit event (see below) without `setTimeout()` seems to work. But, if a submit is done i.e through jquery without refreshing the page, then the submit lock will persist. Or without hitting at all the submit event.

    $("form").submit(function (e) {
        if (!!this.submitted) {
            e.preventDefault();
            return;
        }
        this.submitted = true;
    });

So, maybe better to filter by using the click event of the submit button as before. And the only way i've found to fix the issue is by using `setTimeout()`.

    $("form button[type='submit'], form input[type='submit']").click(function (e) {
        var form = $(this).closest("form")[0];
        if (!!form.submitted) {
            e.preventDefault();
            return;
        }
        form.submitted = true;
        setTimeout(function () {
            form.submitted = false;
        }, 1000);
    });

Best.

**Pull request closed without merging**